ramalama/cli.py

```diff
@@ -238,6 +240,58 @@ def configure_subcommands(parser):
     version_parser(subparsers)
 
 
+def show_gpus_available_cli(args):
```

Comment:
unless I'm mistaken, this appears to not be registered via `parser.add_argument`

Reply:
@maxamillion we don't use --show-gpus-available anymore (@rhatdan requested, if request for me is priority ;-) ). We are now only using `ramalama info`